### PR TITLE
Update for new version of demo store

### PR DESCRIPTION
**Pull request by Zwyx**
### Recreate the demo store following loads of changes from Shopify

Run `npm create @shopify/hydrogen@latest`

### CODE REVIEW

File: app/entry.server.tsx [hunk 1]
```diff
@@ -9,9 +10,38 @@ export default async function handleRequest(
...
+      'https://d3hw6dc1ow8pp2.cloudfront.net',
+      'https://d3g5hqndtiniji.cloudfront.net',
+      'data:',
+    ],
+    styleSrc: [
+      "'self'",
+      "'unsafe-inline'",
+      'https://cdn.shopify.com',
+      'localhost:*',
+      'https://d3hw6dc1ow8pp2.cloudfront.net',
```
Comment (owlyowl) [inline app/entry.server.tsx:27]: Interesting that the merchant will have to need to know to add these.

File: app/routes/collections.$handle.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+          alt={product.featuredImage.altText || product.title}
+          aspectRatio="1/1"
+          data={product.featuredImage}
+          loading={loading}
+          sizes="(min-width: 45em) 400px, 100vw"
+        />
+      )}
+      <h4>{product.title}</h4>
+      <small>
+        <Money data={product.priceRange.minVariantPrice} />
```
Comment (owlyowl) [inline app/routes/collections.$handle.tsx:107]: We probably want to add our star rating snippets to each product on the collection page usually above the money

File: package.json [hunk 3]
```diff
@@ -1,66 +1,55 @@
...
-    "@okendo/shopify-hydrogen": "^2.0.0",
-    "@remix-run/react": "1.14.0",
-    "@shopify/cli": "3.29.0",
-    "@shopify/cli-hydrogen": "^4.0.9",
-    "@shopify/hydrogen": "^2023.1.6",
-    "@shopify/remix-oxygen": "^1.0.4",
-    "clsx": "^1.2.1",
-    "concurrently": "^7.5.0",
-    "cross-env": "^7.0.3",
+    "@okendo/shopify-hydrogen": "file:okendo-shopify-hydrogen-2.0.3.tgz",
```
Comment (owlyowl) [inline package.json:16]: We'll want to change this when the new version launches please

**Review by owlyowl — CHANGES_REQUESTED** [review]
A few minor changes please

File: app/entry.server.tsx (hunk 1 above)
Comment (Zwyx) [reply app/entry.server.tsx:27]: Indeed, but I can't think of another way to do it :pensive:

File: package.json (hunk 3 above)
Comment (Zwyx) [reply package.json:16]: Yep, the commit is marked WIP for this reason.

File: app/routes/collections.$handle.tsx (hunk 2 above)
Comment (Zwyx) [reply app/routes/collections.$handle.tsx:107]: I can't access this page in the demo store, I don't know where it is! I'll merge for now as this is enough for merchants to understand what to do, and we'll see together on Monday about this page, if that's ok with you :wink:

**Pull request merged**